package protoFile.schedule.ObtainTargetReward3204;
message ObtainTargetRewardRequest{
    required int32 id=1;//½ÇÉ«id
    required int32 taskId=2;//ÈÎÎñid
}
message ObtainTargetRewardResponse{
    optional bool result=1;
    optional string message=2;
}